### PR TITLE
Add post-processing script for ort report

### PR DESCRIPTION
**Pull request by Martin-Idel**

### CODE REVIEW

File: octrc-scripts/ort.genPackagesCsv.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
```
Comment (maxhbr) [inline octrc-scripts/ort.genPackagesCsv.py:1]: missing license header ;)

**Review by maxhbr — APPROVED** [review]

**Comment by sschuberth** [comment]
How about contributing a CSV reporter to ORT upstream instead? 😉

**Comment by Martin-Idel** [comment]
@sschuberth Maybe - this was really just a 10 minute effort to get some data to a special format.

File: octrc-scripts/ort.genPackagesCsv.py (hunk 1 above)
Comment (Martin-Idel) [reply octrc-scripts/ort.genPackagesCsv.py:1]: See now below.

**Pull request merged**